I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 387, in script
    scene Zadnik lug
  File "renpy/common/000window.rpy", line 119, in _window_auto_callback
    _window_hide(auto=True)
  File "renpy/common/000window.rpy", line 104, in _window_hide
    renpy.with_statement(trans)
OSError: Couldn't find file 'sounds/Failing.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 387, in script
    scene Zadnik lug
  File "G:\renpy-8.2.1-sdk\renpy\ast.py", line 1242, in execute
    statement_name("scene")
  File "G:\renpy-8.2.1-sdk\renpy\ast.py", line 52, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 119, in _window_auto_callback
    _window_hide(auto=True)
  File "renpy/common/000window.rpy", line 104, in _window_hide
    renpy.with_statement(trans)
  File "G:\renpy-8.2.1-sdk\renpy\exports.py", line 1781, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "G:\renpy-8.2.1-sdk\renpy\display\core.py", line 1548, in do_with
    return self.interact(trans_pause=True,
  File "G:\renpy-8.2.1-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "G:\renpy-8.2.1-sdk\renpy\display\core.py", line 3022, in interact_core
    renpy.audio.audio.periodic()
  File "G:\renpy-8.2.1-sdk\renpy\audio\audio.py", line 1205, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "G:\renpy-8.2.1-sdk\renpy\audio\audio.py", line 1183, in periodic_thread_main
    periodic_pass()
  File "G:\renpy-8.2.1-sdk\renpy\audio\audio.py", line 1125, in periodic_pass
    c.periodic()
  File "G:\renpy-8.2.1-sdk\renpy\audio\audio.py", line 534, in periodic
    topf = load(filename)
  File "G:\renpy-8.2.1-sdk\renpy\audio\audio.py", line 72, in load
    rv = renpy.loader.load(fn, directory="audio")
  File "G:\renpy-8.2.1-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'sounds/Failing.mp3'.

Windows-10-10.0.17763 AMD64
Ren'Py 8.2.1.24030407
SlavSky 1.0
Fri May 17 02:49:35 2024
